docs/source/models.rst

```diff
@@ -139,48 +139,134 @@ For more information, see the :mod:`mlflow.pyfunc` documentation.
 H\ :sub:`2`\ O (``h2o``)
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The H2O model flavor enables logging and loading H2O models. These models will be saved by using the :py:mod:`mlflow.h2o.save_model`. Using :py:mod:`mlflow.h2o.log_model` will also enable a valid ``Python Function`` flavor.
+The ``h2o`` model flavor enables logging and loading H2O models. 
 
-When loading a H2O model as a PyFunc model, :py:mod:`h2o.init(...)` will be called. Therefore, the right version of h2o(-py) has to be in the environment. The arguments given to :py:mod:`h2o.init(...)` can be customized in ``model.h2o/h2o.yaml`` under the key ``init``. For more information, see :py:mod:`mlflow.h2o`.
+The :py:mod:`mlflow.h2o` module defines :py:func:`save_model() <mlflow.h2o.save_model>` and 
+:py:func:`log_model() <mlflow.h2o.log_model>` methods for saving H2O models in MLflow model format.
+These methods produce MLflow models with the ``python_function`` flavor, allowing them to be loaded 
+as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. When MLflow 
+models with the ``h2o`` flavor are loaded via :py:func:`load_pyfunc() <mlflow.pyfunc.load_pyfunc>`, 
+the `h2o.init() <http://docs.h2o.ai/h2o/latest-stable/h2o-py/docs/h2o.html#h2o.init>`_ method is 
+called. Therefore, the correct version of ``h2o(-py)`` must be installed in the loader's 
+environment. The arguments given to 
+`h2o.init() <http://docs.h2o.ai/h2o/latest-stable/h2o-py/docs/h2o.html#h2o.init>`_ can be 
+customized by modifying the ``init`` entry of the persisted H2O model's YAML configuration file: 
+``model.h2o/h2o.yaml``.
+
+Finally, the :py:func:`mlflow.h2o.load_model()` method can be used to load MLflow models with the
+``h2o`` flavor as H2O model objects.
+
+For more information, see :py:mod:`mlflow.h2o`.
 
 Keras (``keras``)
 ^^^^^^^^^^^^^^^^^
 
-The ``keras`` model flavor enables logging and loading Keras models. This model will be saved in a HDF5 file format, via the model_save functionality provided by Keras. Additionally, model can be loaded back as ``Python Function``. For more information, see :py:mod:`mlflow.keras`.
+The ``keras`` model flavor enables logging and loading Keras models. The :py:mod:`mlflow.keras`
+module defines :py:func:`save_model() <mlflow.keras.save_model>` and 
+:py:func:`log_model() <mlflow.keras.log_model>` functions that can be used to save Keras models
```

Comment:
that can be used to save -> that you can use to save

Reply:
Fixed!